### PR TITLE
[docs] React.cloneElement

### PR DESCRIPTION
**Pull request by zpao**

### CODE REVIEW

File: docs/docs/ref-01-top-level-api.md [hunk 1]
```diff
@@ -36,6 +36,19 @@ Create and return a new `ReactElement` of the given type. The type argument can
 html tag name string (eg. 'div', 'span', etc), or a `ReactClass` (created via `React.createClass`).
 
 
+### React.cloneElement
+
+```
+ReactElement cloneElement(
+  ReactElement element,
+  [object props],
+  [children ...]
```
Comment (jsfb) [inline docs/docs/ref-01-top-level-api.md]: we are missing a type on children

Comment (zpao) [inline docs/docs/ref-01-top-level-api.md]: Doesn't have a type, it's var args and can be an array, an element, a string, a number, a bool... which basically maps to ReactNode but that's not really a super well defined type, so meh. We already have it exactly like this for `createElement`

Comment (jsfb) [inline docs/docs/ref-01-top-level-api.md]: if it's a varargs, it shouldn't be inside array brackets, should it?

Comment (zpao) [inline docs/docs/ref-01-top-level-api.md]: its optional.

Comment (jsfb) [inline docs/docs/ref-01-top-level-api.md]: Meh, I prefer incremental correctness over consistency, but sure.  Looks good to me.

**Pull request merged**